### PR TITLE
Add Job Experience page.

### PR DESCRIPTION
**Pull request by LightFelicis**

### CODE REVIEW

File: portfolio/src/main/webapp/job-experience/job-experience.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        <div class="description">
+          I'm part of jury comitee of Junior Olympiad in Informatics.
+          I prepare tasks and create tests set for competitors.
+        </div>
+      </div>
+    </div>
+
+    <div>
+      <div class="img-bubble" id="samsung-bubble">
+        <img src="images/samsung-logo.svg" class="bubble-content"></img>
```
Comment (xmxwx) [inline portfolio/src/main/webapp/job-experience/job-experience.html:32]: nit: in HTML5 it's not correct to close the `` tag.

File: portfolio/src/main/webapp/job-experience/job-experience.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+          Primary School in Warsaw in school year 2018/2019.
+        </div>
+      </div>
+    </div>
+
+    <div class="title" id="exit-button">
+      <a class="button" href="../index.html">Main page</a>
+    </div>
+
+
```
Comment (xmxwx) [inline portfolio/src/main/webapp/job-experience/job-experience.html:73]: nit: there are unnecessary trailing spaces here

File: portfolio/src/main/webapp/job-experience/job-experience.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    </div>
+
+    <div>
+      <div class="img-bubble" id="google-bubble">
+        <img src="images/google-logo.svg" class="bubble-content"></img>
+      </div>
+      <div class="img-bubble-big">
+        <div class="description">
+          I'm Google STEP intern this year! This website is my first
+          assignment : )
```
Comment (xmxwx) [inline portfolio/src/main/webapp/job-experience/job-experience.html:49]: nit: ": )" is missing either a nose or an `&nbsp;`. Using a unicode smiley (🙂) should also be fine.

File: portfolio/src/main/webapp/job-experience/job-experience.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    </div>
+
+    <div>
+      <div class="img-bubble" id="staszic-bubble">
+        <img src="images/staszic-logo.svg" class="bubble-content"></img>
+      </div>
+      <div class="img-bubble-big">
+        <div class="description">
+          I really love teaching!
+          I was conducting Algorithms & Data Structures classes in
```
Comment (xmxwx) [inline portfolio/src/main/webapp/job-experience/job-experience.html:61]: nit: while a free-standing ampersand is unambiguous and thus allowed in HTML5, some people say that it's a good practice is to escape it (`&amp;`) anyway

Comment (LightFelicis) [reply portfolio/src/main/webapp/job-experience/job-experience.html:61]: Done.

File: portfolio/src/main/webapp/job-experience/job-experience.html (hunk 3 above)
Comment (LightFelicis) [reply portfolio/src/main/webapp/job-experience/job-experience.html:49]: Done. Although I'm using ": )" on daily basis 😀

File: portfolio/src/main/webapp/job-experience/job-experience.html (hunk 2 above)
Comment (LightFelicis) [reply portfolio/src/main/webapp/job-experience/job-experience.html:73]: Done.

File: portfolio/src/main/webapp/job-experience/job-experience.html (hunk 1 above)
Comment (LightFelicis) [reply portfolio/src/main/webapp/job-experience/job-experience.html:32]: Done.

**Comment by LightFelicis** [comment]
I merged this branch but I resolved the comments in commit 7b5120eabf00df5efefd95a6ccb6a4a736fe48e4. Sorry for merging it too quickly, I thought you are out of the office yesterday.

**Pull request merged**